src/phoenix/experimental/evals/evaluators.py

```diff
@@ -1,7 +1,59 @@
-from typing import List, Optional
+from dataclasses import dataclass
+from functools import partial
+from typing import Any, List, Mapping, Optional, Protocol, runtime_checkable
 
-from phoenix.experimental.evals import PromptTemplate
-from phoenix.experimental.evals.models import BaseEvalModel
+from phoenix.experimental.evals.models import set_verbosity
+from phoenix.utilities.logging import printif
+
+from .models import BaseEvalModel
+from .templates import PromptTemplate
+
+Record = Mapping[str, Any]
+
+
+class EvaluationResult(Protocol):
+    prediction: str
+
+
+@dataclass
+class ClassificationResult:
+    prediction: str
+
+
+@runtime_checkable
+class Evaluator(Protocol):
+    def evaluate(self, record: Record) -> EvaluationResult:
+        ...
+
+    @property
+    def name(self) -> str:
+        ...
+
+
+class LLMClassifier:
+    def __init__(
+        self,
+        name: str,
+        model: BaseEvalModel,
```

Comment:
thought: need to relax this so that the runner can be configured with a model and this get the model via injection.

Reply:
Addressed in `[8260350](https://github.com/Arize-ai/phoenix/pull/1742/commits/8260350e653e79130cb3971a64228a35140cfe54)`, let me know how that looks @mikeldking